sklearn/utils/_plotting.py

```diff
@@ -244,7 +251,9 @@ def _validate_curve_kwargs(
                 )
 
         curve_kwargs_ = [
-            _validate_style_kwargs({"label": label}, curve_kwargs[fold_idx])
+            _validate_style_kwargs(
+                {"label": label, **default_curve_kwargs_}, curve_kwargs[fold_idx]
+            )
```

Comment:
This is indeed a much better behavior. Please extract it a in a separate PR as a bug fix on its own.